### PR TITLE
Fixed typos and minor spelling errors in the docs

### PR DESCRIPTION
**Pull request by fmalina**
- I am not sure if the <Bitteson tag typo was intentional
- typos in LastMechansimStepBlock and  PendingdHotkeyEmission might indicate mistakes in running code

Ludwig Boltzmann's curse is being lifted 🙌

### CODE REVIEW

File: docs/getting-started/wallets.md [hunk 1]
```diff
@@ -292,14 +292,14 @@ wallet = bt.wallet( name = 'my_coldkey', hotkey = 'my_first_hotkey', path = '~/p
 ```python
 create_coldkey_from_uri(self, uri:str, use_password: bool = True, overwrite:bool = False) -> 'Wallet'
 ```
-Creates coldkey from suri string, optionally encrypts it with the user's inputed password.
+Creates coldkey from uri string, optionally encrypts it with user entered password.
```
Comment (rajkaramchedu) [inline docs/getting-started/wallets.md:295]: This is a commented out section, marked off for removal or reuse somewhere later. I will not review this section of the PR.

File: docs/_components.md [hunk 2]
```diff
@@ -85,9 +85,9 @@ const App = () => (
 ### Show Tabs with Source
 <Tabs>
   <TabItem value="ts" label="TS" default>
-```tsx title="<file />Bitterson/help.ts" link="https://figma.com"
+```tsx title="<file />Bittensor/help.ts" link="https://figma.com"
```
Comment (rajkaramchedu) [inline docs/_components.md:88]: This entire document is hidden, unpublished, by design. This was an experiemental thing back when we were coding up the site elements. Not reviewing rest of this doc.

File: docs/_subnet-pages/subnet-21.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ FileTAO
 
 ## Description
 
-FileTAO implements a novel, multi-layered zero-knowledge interactive proof-of-spacetime algorithm. It cleverly uses Pedersen commitments, random challenges leveraging elliptic curve cryptography, sequential seed-based chained hash verification, and merkle proofs to achieve an efficient, robust, secure, and highly available decetralized storage system on the Bittensor network.
+FileTAO implements a novel, multi-layered zero-knowledge interactive proof-of-spacetime algorithm. It cleverly uses Pedersen commitments, random challenges leveraging elliptic curve cryptography, sequential seed-based chained hash verification, and merkle proofs to achieve an efficient, robust, secure, and highly available decentralized storage system on the Bittensor network.
```
Comment (rajkaramchedu) [inline docs/_subnet-pages/subnet-21.md:16]: This entire _subnet-pages/ directory is hidden, unpublished, by design. This was an experiemental thing back when I was maintaining subnet pages. We are no longer publishing subnet pages. Not reviewing rest of this doc.

File: docs/_subnet-pages/subnet-32.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ we believe that the ability to accurately determine AI-generated text will becom
 
 For example, here are some scenarios where LLM detection is important:
 
-* **For ML-engineers**. Whether you are sourcing training data, developing a foundational LLM, or fine tuning on your own data,
+* **For ML-engineers**. Whether you are sourcing training data, developing a foundational LLM, or fine-tuning on your own data,
```
Comment (rajkaramchedu) [inline docs/_subnet-pages/subnet-32.md:22]: This entire _subnet-pages/ directory is hidden, unpublished, by design. This was an experiemental thing back when I was maintaining subnet pages. We are no longer publishing subnet pages. Not reviewing rest of this doc.

**Review by rajkaramchedu — APPROVED** [review]
Great work! Thanks for catching these.

File: docs/subtensor-nodes/subtensor-storage-query-examples.md [hunk 5]
```diff
@@ -552,7 +552,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     netuid = 1
-    result = substrate.query('SubtensorModule', 'LastMechansimStepBlock', [netuid])
+    result = substrate.query('SubtensorModule', 'LastMechanismStepBlock', [netuid])
```
Comment (rajkaramchedu) [inline docs/subtensor-nodes/subtensor-storage-query-examples.md:555]: Actually the one with the spello is correct. Turns out the developer pushed the function with this typo. `subtensorModule.lastMechansimStepBlock: u64`
See it here https://polkadot.js.org/apps/#/chainstate (Go Developer > Extrinsics and selector subtensorModule and the lastMechansimStepBlock function. So, please revert this change.

File: docs/subtensor-nodes/subtensor-storage-query-examples.md [hunk 6]
```diff
@@ -1136,7 +1136,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     hotkey = Keypair.create_from_uri('//Alice').ss58_address
-    result = substrate.query('SubtensorModule', 'PendingdHotkeyEmission', [hotkey])
+    result = substrate.query('SubtensorModule', 'PendingHotkeyEmission', [hotkey])
```
Comment (rajkaramchedu) [inline docs/subtensor-nodes/subtensor-storage-query-examples.md:1139]: Similarly for this. The function exists with the typo on the chain, as  subtensorModule.pendingdHotkeyEmission: u64

Please revert this change.

**Comment by rajkaramchedu** [comment]
Please see the requested changes.

File: docs/subtensor-nodes/subtensor-storage-query-examples.md (hunk 5 above)
Comment (fmalina) [reply docs/subtensor-nodes/subtensor-storage-query-examples.md]: Ok, thanks. Reverted with a comment

File: docs/getting-started/wallets.md (hunk 1 above)
Comment (fmalina) [reply docs/getting-started/wallets.md:295]: Ok, no difference. Left these with their fixes

**Comment by fmalina** [comment]
Requested changes in place. Thank you for your review.

File: docs/subtensor-nodes/subtensor-storage-query-examples.md [hunk 7]
```diff
@@ -552,7 +552,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     netuid = 1
-    result = substrate.query('SubtensorModule', 'LastMechanismStepBlock', [netuid])
+    result = substrate.query('SubtensorModule', 'LastMechansimStepBlock', [netuid])  # ..nsim [sic] typo on the chain
```
Comment (rajkaramchedu) [inline docs/subtensor-nodes/subtensor-storage-query-examples.md:555]: What is ".. nsim [sic]" in the comment? No one understands what this is. Either remove the entire comment or simply say "Typo on the chain". Same comment for both the commits.

**Review by rajkaramchedu — CHANGES_REQUESTED** [review]

File: docs/subtensor-nodes/subtensor-storage-query-examples.md (hunk 7 above)
Comment (rajkaramchedu) [reply docs/subtensor-nodes/subtensor-storage-query-examples.md:555]: I alerted the dev and they are going to fix it, but we don't have ETA. So having a simple comment like "Typo on the chain" will be useful indeed.

Comment (fmalina) [reply docs/subtensor-nodes/subtensor-storage-query-examples.md:555]: https://en.wikipedia.org/wiki/Sic [sic] is a comment to point out that an error is quoted as is. Many people do understand what it means and it is a correct way to mark out quoted erroneous text used in many languages
